ci_framework/roles/devscripts/tasks/01_prepare_host.yml

```diff
@@ -20,33 +20,19 @@
   tags:
     - bootstrap
     - packages
-  block:
-    - name: Attempt repository enablement using Subscription Manager.
-      ansible.builtin.import_tasks: sub_tasks/11_rhsm.yml
-
-    - name: Attempt repository enablement using yum repo.
-      when: not cifmw_devscripts_rhsm_enabled_repos
-      ansible.builtin.import_tasks: sub_tasks/12_yum_repo.yml
-
-    - name: Install the required packages
-      ansible.builtin.import_tasks: sub_tasks/13_packages.yml
+  ansible.builtin.import_tasks: sub_tasks/11_packages.yml
 
 - name: Performing storage tasks.
   tags:
     - bootstrap
-  ansible.builtin.import_tasks: sub_tasks/14_storage.yml
+  ansible.builtin.import_tasks: sub_tasks/12_storage.yml
 
 - name: Performing network tasks.
   tags:
     - bootstrap
-  ansible.builtin.import_tasks: sub_tasks/15_network.yml
+  ansible.builtin.import_tasks: sub_tasks/13_network.yml
```

Comment:
It's renamed from `15_network` to `13_network` here, however in the code, you're renaming `17_virt_net` to `13_network`. `15_network` is deleted.
Is it the right thing?  Both look very different with the content.

Is it under
> Extra networks managed outside of dev-scripts

Reply:
In the existing `15_network`, we wanted to create the `baremetal` bridge connection as it would be required for deploying the OCP platform. In this patch set, we have moved to `ci_nmstate` which would perform all the network configurations. Hence making it redundant.

https://github.com/openstack-k8s-operators/ci-framework/pull/659/files#diff-3842fc1a95f69487987fb5c15bc61bee69876c479ab4990d0a24a10731db5d9fR25-R30

Hope this clarifies the change.

Reply:
Thanks for the explanation.